File: Tarea7_Python/Usuario.py
class Usuario:
  __id=0
  __nombre=""
  __edad=0

  def __init__(self, nombre, id, edad):
    self.__nombre = nombre
    self.__id = id
    self.__edad = edad
    self.__libros_rentados = []

  def get_nombre(self):
    return self.__nombre

  def get_id(self):
    return self.__id

  def get_edad(self):
    return self.__edad

  def get_libros_rentados(self):
    for libro in self.__libros_rentados:
        print("Título del libro:", libro.get_nombre())
        print("Autor:", libro.get_autor())
        print("ID:", libro.get_id())
    return ""

  def get_datos(self):
    print("Nombre:", self.get_nombre())
    print("ID:", self.get_id())
    print("Edad:", self.get_edad())
    return ""

  def add_libro(self, libro):
    self.__libros_rentados.append(libro)

File: Tarea8/src/Usuario.java
import java.util.ArrayList;
public class Usuario {
    private String nombre;
    private int id;
    private int edad;


    private ArrayList<Libro> librosRentados;

    public Usuario(String nombre, int id, int edad){
        this.nombre=nombre;
        this.edad=edad;
        this.id=id;
        this.librosRentados = new ArrayList<>();
    }

    public String getNombre() {
        return nombre;
    }

    public int getId() {
        return id;
    }

    public int getEdad() {
        return edad;
    }

    public String getLibrosRentados() {
        for (Libro libro : librosRentados){

        System.out.println("Título del libro: "+libro.getNombre());
        System.out.println("Autor: "+libro.getAutor());
        System.out.println("ID: "+libro.getId());}
        return ("");
    }

    public String getDatos(){
        System.out.println("Nombre:"+getNombre());
        System.out.println("ID: "+getId());
        System.out.println("Edad: "+getEdad());
        return ("");
    }
    public void addLibro(Libro libro){
        librosRentados.add(libro);
    }
}